### PR TITLE
fortify-on-demand-mobile-assessment 0.0.1

### PR DESCRIPTION
**Pull request by jeremy-chua**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by jeremy-chua** [comment]
how do i get this reviewed?

**Comment by Janos-vrancsik** [comment]
Hi @jeremy-chua, the team will review it shortly. Thanks for your patience! 🙏

File: steps/fortify-on-demand-mobile-assessment/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+support_url: https://github.com/jeremy-chua/bitrise-step-fortify-on-demand-mobile-assessment/issues
+published_at: 2021-09-13T06:20:54.462600745+08:00
+source:
+  git: https://github.com/jeremy-chua/bitrise-step-fortify-on-demand-mobile-assessment.git
+  commit: eca704aa91e6751d5d0ab71e0e4eeb519672dfa0
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
```
Comment (koral--) [inline steps/fortify-on-demand-mobile-assessment/0.0.1/step.yml:17]: Why not other mobile multiplatform types?

File: steps/fortify-on-demand-mobile-assessment/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+- android
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/jeremy-chua/bitrise-step-fortify-on-demand-mobile-assessment
+deps:
+  brew:
+  - name: git
```
Comment (koral--) [inline steps/fortify-on-demand-mobile-assessment/0.0.1/step.yml:26]: Those deps does not seem to be used

File: steps/fortify-on-demand-mobile-assessment/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+is_skippable: false
+run_if: ""
+inputs:
+- client_id: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Client ID for authentication
+    title: Client ID
+- client_secret: null
```
Comment (koral--) [inline steps/fortify-on-demand-mobile-assessment/0.0.1/step.yml:42]: is_sensitive should be set for this input

**Comment by jeremy-chua** [comment]
should i resubmit for a new version "0.0.2" or can i maintain the same version?

**Comment by jeremy-chua** [comment]
hi @koral-- 
i have made some changes from the feedback.:

- Removed dep
- Make client id and secret as sensitive

Please kindly review.

**Pull request closed without merging**